src/DssSpell.sol

```diff
@@ -111,89 +84,98 @@ contract DssSpellAction is DssAction {
-        DssExecLib.sendPaymentFromSurplusBuffer(FEEDBLACK_LOOPS_WALLET, 11_970);
-        DssExecLib.sendPaymentFromSurplusBuffer(ULTRASCHUPPI_WALLET,    11_844);
-        DssExecLib.sendPaymentFromSurplusBuffer(FLIPSIDE_CRYPTO_WALLET, 11_392);
-        DssExecLib.sendPaymentFromSurplusBuffer(PENN_BLOCKCHAIN,         9_773);
-        DssExecLib.sendPaymentFromSurplusBuffer(GFX_LABS_WALLET,         8_512);
-        DssExecLib.sendPaymentFromSurplusBuffer(CHRIS_BLEC,              8_090);
-        DssExecLib.sendPaymentFromSurplusBuffer(ACRE_INVEST_WALLET,      6_681);
-        DssExecLib.sendPaymentFromSurplusBuffer(MHONKASALO_TEEMULAU,     4_492);
-        DssExecLib.sendPaymentFromSurplusBuffer(LLAMA,                   3_797);
-        DssExecLib.sendPaymentFromSurplusBuffer(MAKERMAN_WALLET,         2_670);
-        DssExecLib.sendPaymentFromSurplusBuffer(BLOCKCHAIN_COLUMBIA,     1_809);
-        DssExecLib.sendPaymentFromSurplusBuffer(CODEKNIGHT,                208);
-        DssExecLib.sendPaymentFromSurplusBuffer(FRONTIER_RESEARCH,         207);
-
-        // ------------------ PPG - Maker Open Market Committee ------------------
-        // https://vote.makerdao.com/polling/QmXHnn2u#poll-detail
-
-        ////// Stability Fee Changes //////
-
-        // Decrease the CRVV1ETHSTETH-A Stability Fee from 2.25% to 2.0%.
-        DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A", TWO_PCT_RATE, true);
-
-        // Decrease the MANA-A Stability Fee from 6% to 4.5%.
-        DssExecLib.setIlkStabilityFee("MANA-A", FOUR_FIVE_PCT_RATE, true);
-
-        // Decrease the ETH-A Stability Fee from 2.25% to 1.5%.
-        DssExecLib.setIlkStabilityFee("ETH-A", ONE_FIVE_PCT_RATE, true);
+        // https://mips.makerdao.com/mips/details/MIP40c3SP74
+        // DAIF-001 | 2022-10-01 to 2022-10-31 | 67,863 DAI
+        VEST.restrict(
+            VEST.create(
+                DAIF_WALLET,                                             // usr
+                67_863 * WAD,                                            // tot
+                OCT_01_2022,                                             // bgn
+                OCT_31_2022 - OCT_01_2022,                               // tau
+                0,                                                       // eta
+                address(0)                                               // mgr
+            )
+        );
 
-        // Decrease the ETH-B Stability Fee from 3.75% to 3.0%.
-        DssExecLib.setIlkStabilityFee("ETH-B", THREE_PCT_RATE, true);
+        // https://mips.makerdao.com/mips/details/MIP40c3SP74
+        // DAIF-001 | 2022-11-01 to 2023-08-31 | 329,192 DAI
+        VEST.restrict(
+            VEST.create(
+                DAIF_WALLET,                                             // usr
+                329_192 * WAD,                                           // tot
+                OCT_31_2022,                                             // bgn
```

Comment:
Shouldn't this be November 1st?
(I know it's a second apart but just aligning to the other dates which are the same as specified).

Reply:
Yeah, that was the comment I made.  If you guys think I should change this, I will, but the real time is just a second later and I thought I would save the gas.

Reply:
Yeah, I would change it.
I think it makes it easier to audit and leaves less margin for error if we align stuff perfectly to the sheet.

Reply:
fixed